safe_transaction_service/history/tests/test_views.py

```diff
@@ -2212,6 +2258,17 @@ def test_delete_safe_delegate(self):
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
         self.assertEqual(SafeContractDelegate.objects.count(), 0)
 
+    def _get_module_transaction_id(self, transaction_hash, trace_address):
```

Comment:
I think we shouldn't use these helper methods here. I would hardcode what are we expecting with some predefined `txHash`, as if we are having an implementation issue on the `id` calculation we will not be able to detect it as we are never testing the spec we are defining, for example in this case `{i}{64 hex chars with 0x}{stringfied trace_address list}`